R/helpers.R

```diff
@@ -358,6 +358,9 @@ printVignette <- function (x, ...)
 
 
 ##' A Reference Class to help programming validation for new S4 classes
+##' 
+##'
+##' @description `r lifecycle::badge("experimental")`
```

Comment:
as mentioned please use 
```suggestion
##' @description `r lifecycle::badge("stable")`
```

Reply:
Done, thanks!